### PR TITLE
Create "Bsn" rule

### PR DESCRIPTION
**Pull request by rdrenth**

### CODE REVIEW

File: tests/unit/Rules/BsnTest.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @group  rule
+ * @covers Respect\Validation\Rules\BsnSet
```
Comment (henriquemoody) [inline tests/unit/Rules/BsnTest.php]: I guess that's why Travis is failing

**Comment by rdrenth** [comment]
Oops, my bad on the wrong @covers tag in the test, fixed it now!

**Comment by henriquemoody** [comment]
This rule will be available on Oct 24th along with version 1.0.

Thanks for contributing! :panda_face:

**Comment by rdrenth** [comment]
Awesome! Thanks :smile:

**Pull request merged**